server/src/main/java/io/crate/rest/action/RestBulkRowCountReceiver.java

```diff
@@ -46,13 +46,19 @@ class RestBulkRowCountReceiver extends BaseResultReceiver {
     @Nullable
     public CompletableFuture<Void> setNextRow(Row row) {
         rowCount = (long) row.get(0);
-        // Can be an optimized bulk request with only 1 bulk arg/operation which carries only 1 column (row count).
-        if (results.length > 1) {
+        try {
             failure = (Throwable) row.get(1);
+        } catch (IndexOutOfBoundsException e) {
+            // ignore, must be an optimized bulk request with only 1 bulk arg/operation
```

Comment:
This change prevents the optimized bulk request with 1 bulk arg. I think we can revert https://github.com/crate/crate/pull/16921.